ci/setup_env.sh

```diff
@@ -12,8 +14,13 @@ if [[ "$(uname)" == "Linux" && -n "$LC_ALL" ]]; then
     echo
 fi
 
-MINICONDA_DIR="$HOME/miniconda3"
 
+if [ `uname -m` = 'aarch64' ]; then
+   MINICONDA_DIR="$HOME/archiconda3"
+   IS_SUDO="sudo"
+else
+   MINICONDA_DIR="$HOME/miniconda3"
```

Comment:
you can get rid of this, the directory is fine even if it's not miniconda in this case

Reply:
can you please address this comment